### ISSUE TITLE
Table-backend migration generator creates wrong migrations

### ISSUE THREAD

**Issue by omitter** [shioyama/mobility#281]
Running for instance this command  
`rails generate mobility:translations customer_notification content:string`  
generates a migration file containing this malformed line
```ruby
      t.integer :customer_notification_id, null: false
```

As far as I understand, this should be:
```ruby
      t.references :customer_notification, null: false, foreign_key: true
```

Rails 5.1 changed the default id column type from `integer` to `bigint`, so translation and model columns don't match up anymore. As there is no foreign key, it's not noticed though. As long as we don't hit the `integer` limit there is no problem I guess, but if, I guess it blows. `t.references` automatically generates the right column type (bigint for ActiveRecord::Migration[5.1] and above, integer for ActiveRecord::Migration[5.0] I guess).

Also, is there any reason not to use a foreign key? I guess everybody wants the database to check for consistency, so I added that in the command above.

**Comment by shioyama** [comment]
Thanks for noticing this, and for making a quick PR to fix it. I'll respond on the PR.

**Comment by shioyama** [comment]
The fix for this was released in 0.8.1.
